### ISSUE TITLE
cover only_metadata,tex

### ISSUE THREAD

**Issue by jbs1** [jbs1/tract2#130]
_migrated from Trac, where originally posted by **kohlhase** on 2-May-2013 8:30am_

Christian Lange has supplied us an example file `examples/only_metadata` that is devoid of any context. We need to cover this, which involves covering the metadata treatment in `beilstein.cls`. 

Most of that can probably be adapted from `LaTeX.pool.ltxml` so that we have the information in `*.ltxml` and can base the beilstein xml on that. This ticket only concerns the first step: the `beilstein.cls.ltxml` binding (second step is #1478). Deyan will supervise this.

**Comment by jbs1** [comment]
_migrated from Trac, where originally posted by **kohlhase** on 2-May-2013 4:05pm_

(In #1478) I made a rule in the Makefile to make `*.beil` (Beilstein XML files), e.g. `only_metadata.beil` (committed). Lukas, you just need to add templates to `ltx2beil.xsl`. BTW, I made a convenient fallback template that gives you error messages which templates are still unimplemented. 

To learn about the target format look at `resources/beilstein_beispiel.xml` and `resources/Creating_Beilstein_XMLFile.pdf`.
